tests/functional/object/mcg/test_s3_vector.py

```
@@ -1310,3 +1331,592 @@ def test_vector_distance_metrics(self, vector_bucket_factory, mcg_obj):
+            s3vectors_client,
+            [
+                {
+                    "key": target_key,
+                    "data": {"float32": vectors_a[0]["data"]["float32"]},
+                    "metadata": updated_metadata,
+                }
+            ],
+            vectorBucketName=vector_bucket_name,
+            indexName=index_name,
+        )
+        logger.info(f"✓ Updated '{target_key}' with new metadata: {updated_metadata}")
```

Ruff (0.15.18)

[warning] Logging statement uses f-string

(G004)